modules/Reports/templates_assets.php

```diff
@@ -37,9 +38,31 @@
     $prototypeGateway = $container->get(ReportPrototypeSectionGateway::class);
     $fontGateway = $container->get(ReportTemplateFontGateway::class);
 
+    $request = ['active' => $_GET['active'] ?? 'Y'];
```

Comment:
Although we don't need this form for the filters anymore, the asset page desperately needs a search function. Are you able to change the active filter into a search field, which should be easy to connect to the existing DataTable. Thanks!